pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -363,70 +362,68 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(ctx context.Context, request re
 	return reconcile.Result{}, nil
 }
 
-// configureNetPermissionsForFileVolume helps to add or remove net permissions for
-// a given file volume. The callers of this method can remove or add net permissions
-// by setting the parameter removePermission to true or false respectively.
-// Returns error if any operation fails
-func (r *ReconcileCnsFileAccessConfig) configureNetPermissionsForFileVolume(ctx context.Context, volumeID string, vm *vmoperatortypes.VirtualMachine, instance *cnsfileaccessconfigv1alpha1.CnsFileAccessConfig, removePermission bool) error {
+// configureNetPermissionsForFileVolume helps to add or remove net permissions
+// for a given file volume. The callers of this method can remove or add net
+// permissions by setting the parameter removePermission to true or false
+// respectively. Returns error if any operation fails.
+func (r *ReconcileCnsFileAccessConfig) configureNetPermissionsForFileVolume(ctx context.Context,
+	volumeID string, vm *vmoperatortypes.VirtualMachine, instance *cnsfileaccessconfigv1alpha1.CnsFileAccessConfig,
+	removePermission bool) error {
 	log := logger.GetLogger(ctx)
 	tkgVMIP, err := r.getVMExternalIP(ctx, vm)
 	if err != nil {
-		msg := fmt.Sprintf("Failed to get external facing IP address for VM: %s/%s instance. Error: %+v", vm.Namespace, vm.Name, err)
-		log.Error(msg)
-		return errors.New(msg)
+		return logger.LogNewErrorf(log, "Failed to get external facing IP address for VM: %s/%s instance. Error: %+v",
```

Comment:
Not related to this change - but there are many error strings in this file that start with a capital letter, which we try to avoid. Will leave it up to you if you want to fix this in the current PR.

Reply:
Yes, Raunak, thanks for pointing this out. Sandeep, Xing, myself, and a few others had a similar discussion over past changes. Our error messages are not consistent. We probably need dedicated efforts from code owners to purify them. So, I am using the same error messages as before (without regression). Sandeep (?) had filed a JIRA ticket to follow up on this.